src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $card-width);
+  gap: $space;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $card-width);
+    margin: 0 auto;
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $card-width);
+  }
+
   &__card {
-    display: flex;
+    box-sizing: border-box;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
```

Comment:
your margin under the table totally disappeared, add margin-bottom for catalog grid